bug_bounty.md

```diff
@@ -0,0 +1,71 @@
+
+As above, please DO NOT include attachments or provide detail regarding the security issue in this email.
+
+# Petra Bug Bounty
+
+Aptos Labs offers bounties for security reports. Reports will be validated against the current released version of the [Petra wallet](https://chrome.google.com/webstore/detail/petra-aptos-wallet/ejjladinnckdgjemekebdpeokbikhfci) and no others. This includes only features that have been explicitly enabled by the team and not those that might have been toggled on by modifying debug flags, etc.
+
+Aptos Labs considers the following levels of severity:
+
+****************Critical — Up to $100,000 USD in APT tokens (locked for 12 months)****************
+
+- Website can retrieve plaintext (decrypted) sensitive data or files from the wallet (excluding non-sensitive environment variables, open source code, or usernames), such as plaintext private keys, plaintext sensitive passwords, plaintext recovery phrases.
+- Taking state-modifying authenticated actions without any interaction by that user via websites (not via an user’s unlocked computer), such as signing arbitrary bytes, and signing/submitting malicious transactions without user knowledge.
+- Malicious interactions with the wallet, without user knowledge, such as modifying transaction arguments or parameters, substituting contract addresses, and being able to extract the private keys without user consent.
+- Injection of malicious HTML or XSS into the wallet, via injected DApp API, DApp controlled error messages, and NFT metadata.
+
+**Other Issues**
+
+We are working on a process to accept non-critical bugs that result in heavy negative impact to the user experience.
+
+## **Report Details**
+- Summary of the issue
+- Does this relate the definition of a critical issue as defined above? Which point?
+- Clear and complete steps to repro the issue
+- If applicable, proposed solution
+
+## **Payment of Bug Bounties**
+
+- Bounties are currently awarded on a rolling/weekly basis and paid out within 30 days upon receipt successful KYC and payment contract.
+- The APT/USD conversion rate used for payments is the market price of APT (denominated in USD) at 11:59 PM PST the day that both KYC and the payment contract are completed.
+- The reference for this price is the Closing Price given by Coingecko.com on that date given here: [https://www.coingecko.com/en/coins/aptos/historical_data#panel](https://www.coingecko.com/en/coins/aptos/historical_data#panel)
+- Bug bounties that are paid out in APT are locked to the account provided by the reporter with a lockup expiring 12 months from the date of the delivery of APT.
+
+## Duplicate Reports
+
+Compensation for duplicate reports will be split among reporters with the first to report taking priority using the following equation:
+
+```
+R: total reports
+ri: report priority
+bi: bounty share
+
+bi = 2 ^ (R - ri) / ((2^R) - 1)
```

Comment:
$$b_i = \frac{2^{R - r_i}}{2^R - 1}$$

Reply:
We could add latex straight in so it's clearer?

Reply:
will do